### PR TITLE
Complete SSO flow in popup without the need for user to click  the start button

### LINKED ISSUE #99: Popup does not open after SSO has completed
Once we're back to the page after SSO, the block should open automatically without the user clicking the button, so that login can be completed.

### PR DESCRIPTION
**Pull request by psrpinto**
Fixes #99

I also did a bit of a refactor in order to have an `IframeUrl` class, which can be used directly by the block, since the block renders the markup in JSX, it does not need the logic in `Iframe`.

----

https://user-images.githubusercontent.com/550401/199987615-a060f621-7ac5-42c9-838e-2fadbb93b186.mov

### CODE REVIEW

**Comment by psrpinto** [comment]
Sure thing @akirk, updated the PR description.

File: frontend/parent/iframe.ts [hunk 1]
```diff
@@ -1,32 +1,55 @@
...
+                paramName,
+                queryParams.get(paramName) ?? ""
+            );
+        }
     }
+}
 
-    public get visible(): boolean {
-        return this.iframe.style.display !== "none";
+export class Iframe {
```
Comment (akirk) [inline frontend/parent/iframe.ts:46]: Is it ok to have two classes in a file? (Maybe I am just spoiled by WordPress coding standards)

**Review by akirk — APPROVED** [review]
Looks great, nice!

File: frontend/parent/iframe.ts (hunk 1 above)
Comment (psrpinto) [reply frontend/parent/iframe.ts:46]: I actually considered moving `IframeUrl` to its own file, but the file would be called `iframe-url.ts` which didn't feel right :D That said, it is somewhat common, at least in the hydrogen codebase, to have related classes in the same file.

I think we should rename all files to be `CamelCase.ts`, and then extract each class to its own file.

**Pull request merged**